Assets/Scripts/Scriptables/Systems/DamageResistanceSystem.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Celeritas.Game;
+using Celeritas.Game.Entities;
+using Celeritas.Scriptables.Interfaces;
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace Celeritas.Scriptables.Systems
+{
+
+	/// <summary>
+	/// Reduces incoming damage by set percentage from all sources.
+	/// </summary>
+
+	[CreateAssetMenu(fileName = "New Ship Damage Resistance Modifier", menuName = "Celeritas/Modifiers/Ship Damage Resistance")]
+
+	public class DamageResistanceSystem : ModifierSystem, IEntityEffectAdded, IEntityEffectRemoved
+	{
+
+		[SerializeField, PropertyRange(-100, 100), Title("Damage Taken Modifier Percentage")]
+		private int amount;
+
+		[SerializeField, PropertyRange(-100, 100), Title("Damage Taken Modifier Percentage added per level")]
+		private int amountExtraPerLevel;
+
+		/// <summary>
+		/// The amount this system modifies damage taken
+		/// </summary>
+		public int Amount { get => amount; }
+
+		/// <summary>
+		/// How much the damage taken modifier increases per level
+		/// </summary>
+		public int AmountExtraPerLevel { get => amountExtraPerLevel; }
+
+		public override bool Stacks => true;
+
+		public override SystemTargets Targets => SystemTargets.Ship;
+
+		public override string GetTooltip(ushort level) => $"<color=green>▲</color> Reduces amount of damage taken by <color=green>{(Amount + (AmountExtraPerLevel * level)):0}%</color>.";
```

Comment:
Tiny thing, but there's been an update and I don't think we are using the little arrows anymore. If you playtest and look at the module descriptions by hovering your mouse over them, you'll see what I mean.